py/picca/data.py

```diff
@@ -63,25 +63,25 @@ class forest(qso):
 
     def __init__(self,ll,fl,iv,thid,ra,dec,zqso,plate,mjd,fid,order):
         qso.__init__(self,thid,ra,dec,zqso,plate,mjd,fid)
-        ## rebin
 
+        ## rebin
         bins = ((ll-forest.lmin)/forest.dll+0.5).astype(int)
         w = bins>=0
         fl=fl[w]
         iv =iv[w]
         bins=bins[w]
 
```

Comment:
let's take the opportunity to clean up this code a bit. Can you add a cut on forest.lmin and forest.lmax earlier? 
e.g.:

    w = (ll>forest.lmin) & (ll<forest.lmax)
    ll = ll[w]
    fl = fl[w]
    iv = iv[w]

    bins = ((ll-forest.lmin)/forest.dll+0.5).astype(int)

etc...

Reply:
We can't remove it from after rebining since sometime the last bin has its original ll<lmax, but its rebined one ll>lmax. However, we can add cutting also before rebining. Do you want this ?

Reply:
right, in the snippet of code above, the cut is before the definition of `bins = ...`, so yes, before rebinning. Thanks!